psh/test-touch-rootfs.py

```diff
@@ -0,0 +1,33 @@
+# Phoenix-RTOS
+#
+# phoenix-rtos-tests
+#
+# psh touch command test for rootfs targets
+#
+# Copyright 2022 Phoenix Systems
+# Author: Damian Loewnau
+#
+# This file is part of Phoenix-RTOS.
+#
+# %LICENSE%
+
+import psh.tools.psh as psh
+from psh.tools.common import assert_mtime_change
+
+
+def assert_hardlinks(p):
+    dates = {}
+    # all psh commands are hardlinks
+    psh_hardlinks = psh.get_commands(p)
+    for hardlink in psh_hardlinks:
+        hardlink_path = f'/bin/{hardlink}'
+        dates[hardlink] = psh.date(p)
+        msg = f"Prompt hasn't been seen after the hardlink touch: {hardlink_path}"
+        psh.assert_cmd(p, f'touch {hardlink_path}', '', msg)
```

Comment:
If we need basename then I would not define new variable and keep just "/bin" prefix, but can be as is :).
Also don't need to pass empty string, just use `msg=msg`

Reply:
I would rather leave it then.